sdk/mgmtcommon/AppAuthentication/Azure.Services.AppAuthentication/TokenProviders/MsiAccessTokenProvider.cs

```diff
@@ -106,7 +111,8 @@ internal MsiAccessTokenProvider(HttpClient httpClient, int retryTimeoutInSeconds
                 // Craft request as per the MSI protocol
                 var requestUrl = isAppServicesMsiAvailable
                     ? $"{msiEndpoint}?resource={resource}{clientIdParameter}&api-version=2019-08-01"
-                    : $"{AzureVmImdsEndpoint}?resource={resource}{clientIdParameter}&api-version=2018-02-01";
+                    : $"{AzureVmImdsEndpoint}{ImdsTokenRoute}?resource={resource}{clientIdParameter}" +
```

Comment:
single string interpolation is easier to read and more efficient

Reply:
Went with multiline because of line-length concerns, but that's a fair point.